specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,104 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer, a decimal value, or a value represented in scientific notation as defined in this section.
```

Comment:
Scientific notation is not compute friendly…. I would prefer we don’t allow this. Thoughts?

Reply:
Yeah, I could go both ways.  I think I'm leaning towards us not stating anything about scientific notation.   I just added this "just in case," but I don't see a need for it.

@ljadvey I'm going to take it out of this request - it's a distraction.

Reply:
Just to make it clear, I don't know if I'm going to expressly prohibit it, but I think we just leave it out.    From a practical perspective, if I have floating point numbers, every language I know of is going to know how to parse them, but I just don't think we need to state this one way or the other.

The only place I can think of where something like this is useful, is when we have several underutilized storage  - if you are being charged in PBs, but you are only consuming a fraction of a percent of that unit, you end up with numbers like 0.000000342.   But, I don't think it's worth diving into this level of specificity on the format